.travis.yml

```diff
@@ -26,6 +26,21 @@ jobs:
           cd ./tests/phpcs &&
           composer install &&
           ./vendor/bin/phpcs --standard=phpcs.xml
+    - name: Seeders tests
+      script: |
+        ./run.sh & \
+        /bin/bash -c "
+          until docker-compose -p track exec web curl localhost
```

Comment:
What is the point? To make blocking loop until the process starts? I think there should be timeout. Something like this:

```sh
timeout 60 /bin/bash -c 'until until docker-compose -p track exec web curl localhost; do echo "Waiting for web server...'"; sleep 3; done'
````